pkg/shp/cmd/buildrun/create.go

```diff
@@ -60,6 +62,13 @@ func (c *CreateCommand) Run(params *params.Params, ioStreams *genericclioptions.
 		},
 		Spec: *c.buildRunSpec,
 	}
+
+	envs, err := c.cmd.Flags().GetStringArray("env")
```

Comment:
Why do we need to copy it back to the `BuildRun` instance? I think it should be using the `BuildRun` instance wired by `flags` package, thus it should have the `env` slice already populated.

Reply:
The flag can not have a type of corev1.EnvVar, so the array of strings needs to be processed into that type.